packages/pangea-node-sdk/src/services/authn.ts

```diff
@@ -0,0 +1,421 @@
+import PangeaResponse from "../response";
+import BaseService from "./base";
+import PangeaConfig from "../config";
+import { AuthN } from "../types";
+
+import { schema } from "../utils/validation";
+/**
+ * AuthnService class provides methods for interacting with the AuthN Service
+ * @extends BaseService
+ */
+class AuthnService extends BaseService {
+  constructor(token: string, config: PangeaConfig) {
+    super("authn", token, config);
+    this.apiVersion = "v1";
+  }
```

Comment:
Instead of needing to override the constructor, it may be nicer just to have "authn" and apiVersion as properties, I mean apiVersion already is one, and I don't think you need to set it in the constructor should just be able to override the property on the class definition

Reply:
Actually I see serviceName is a property, ya I guess I don't get why is this set through the constructor, versus just on class definition

Reply:
I wrote it like this because it was how all the other service classes are being written. If we'd want to change it, I would push for handling that in a separate pull request.